### PR TITLE
Mol 17278

### PR DESCRIPTION
**Pull request by meman**
TICKET - http://jira.mol.dmgt.net/browse/MOL-17278

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -1,5 +1,11 @@
 'use strict';
```
Comment (meman) [inline Gruntfile.js]: All changes here are to allow es6 code

**Review by sielay — DISMISSED** [review]

File: lib/components/ComboList.js [hunk 2]
```diff
@@ -130,11 +140,13 @@ function onItemsChange(msg, data) {
 }
 
 function MLComboList_get() {
-    var value = this.model.get();
+    const value = this.model.get();
     return (value && typeof value === 'object') ? _.clone(value) : value;
 }
 
 function MLComboList_set(value) {
+    const hasTooltips = value && Array.isArray(value) && !!value.find(v => v.item);
```
Comment (jacdebug) [inline lib/components/ComboList.js]: is check value redundant as Array.isArray can catch undefined, if that is the case.

File: lib/components/SuperCombo.js [hunk 3]
```diff
@@ -300,6 +309,9 @@ function MLSuperCombo$update() {
 
     var arrToShow = this._filteredOptionsData.slice(this._startIndex, this._endIndex);
 
+    const hasTooltips = arrToShow && Array.isArray(arrToShow) && !!arrToShow.find(v => v.item);
```
Comment (jacdebug) [inline lib/components/SuperCombo.js]: same as above
is check value redundant as Array.isArray can catch undefined, if that is the case.

**Review by jacdebug — DISMISSED** [review]
one comment

File: lib/components/ComboList.js (hunk 2 above)
Comment (meman) [reply lib/components/ComboList.js]: fair happy to change

**Review by sielay — DISMISSED** [review]

**Review by jacdebug — DISMISSED** [review]

File: lib/components/ComboList.js [hunk 4]
```diff
@@ -77,6 +77,8 @@ function MLComboList$setDataValidation(dataValidation) {
 }
 
 function MLComboList$setOptions(options) {
+    const hasTooltips = Array.isArray(options) && !!options.find(v => v.item);
```
Comment (jacdebug) [inline lib/components/ComboList.js:80]: `v` ? better be with a readable variable name always IMHO

File: lib/components/SuperCombo.js [hunk 5]
```diff
@@ -309,7 +309,7 @@ function MLSuperCombo$update() {
 
     var arrToShow = this._filteredOptionsData.slice(this._startIndex, this._endIndex);
 
-    const hasTooltips = arrToShow && Array.isArray(arrToShow) && !!arrToShow.find(v => v.item);
+    const hasTooltips = Array.isArray(arrToShow) && !!arrToShow.find(v => v.item);
```
Comment (jacdebug) [inline lib/components/SuperCombo.js:312]: same as above for `v`

**Review by jacdebug — APPROVED** [review]
Looks good, one comment for next time please.

**Pull request merged**